apps/condo/domains/ticket/utils/serverSchema/resolveHelpers.js

```diff
@@ -282,8 +286,41 @@ async function classifyTicket (context, details) {
     })
 }
 
+const STATUS_RELATED_TICKET_FIELDS = ['status', 'statusUpdatedAt', 'deferredUntil']
+
+const createOfflineTicketStatusChange = async ({ schema, operation, context, existingItem, resolvedData }) => {
+    await storeChangesIfUpdated(
+        pick(schema.fields, ['status']),
+        createTicketChange({ isOfflineChange: true, actualCreationDate: resolvedData.statusUpdatedAt }),
+        ticketChangeDisplayNameResolversForSingleRelations,
+        relatedManyToManyResolvers,
+    )({ operation, existingItem: { id: existingItem.id }, context, updatedItem: { sender: existingItem.sender, status: resolvedData.status } })
```

Comment:
`deferredUntil` won't be set if payload have it?

Reply:
fixed